.github/workflows/run-network-tests.yml

```
@@ -14,7 +14,7 @@ jobs:
       matrix:
         node-version: [14.x]
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
```

I think this should be `v3` to match the other updates:
```suggestion
    - uses: actions/checkout@v3
```